Features/Lookuptable/testPages/data/lookup_table_page.py

```diff
@@ -0,0 +1,771 @@
+        self.expression_editor =(By.XPATH,"//*[@class='fd-scrollable full']")
+        self.appname =(By.XPATH,"//*[@class='variable-app_name appname-text']")
+        self.makenewbuild =(By.XPATH,"//*[@id='releases-table']/p[2]/button")
+        self.release = (By.XPATH, "//*[@id='releases-table']/div[6]/div[1]/div[1]/div/div/button[1]")
+        self.login_user= (By.XPATH,"//*[@class='fa fa-user appicon-icon']")
+        self.select_user= "//*[@aria-label='{}']"
+        self.login = (By.XPATH,"//*[@id='js-confirmation-confirm']")
+        self.start= (By.XPATH,"//*[@class='ff ff-start-fg appicon-icon appicon-icon-fg']")
+        self.inapp_caselist = (By.XPATH,"//*[@class='formplayer-request']")
+        self.inapp_registrationform = (By.XPATH,"//h3[text()='Registration Form']")
+        self.inapp_selectoption = (By.XPATH,"//*[(@class='sel clear')]//div/div[1]/label/span/p")
+        self.inapp_next = (By.XPATH,"//*[@class='btn btn-formnav btn-formnav-next'][1]")
+        self.inapp_continue = (By.XPATH,"//*[@class='btn btn-success btn-formnav-submit'][1]")
+        self.inapp_success_message = (By.XPATH,"//*[@class='alert alert-success']")
+        self.caselist= (By.XPATH,"//*[@aria-label='Case List']")
+        self.registrationform= (By.XPATH,"//*[@aria-label='Registration Form']")
+        self.preview= (By.XPATH,"//*[@class='preview-toggler js-preview-toggle']")
+        self.confirm = (By.XPATH,"//*[contains(text(),'Yes, log in as this user')]")
+        self.SPECIFIC_Registration_form = (By.XPATH,"/html/body/div[1]/div[4]/div/div[1]/nav/ul[1]/li/ul/li[1]/div/a[2]")
+        self.revist_lookup_tabble = (By.XPATH,"//*[@class='fd-scrollable fd-scrollable-tree']/div/ul/li/ul/li/a")
+        self.valueerror = (By.XPATH,"//*[@class='fd-scrollable fd-props-scrollable']/form/fieldset/div/div/div[2]/div/div/div/div")
+        self.displayerror = (By.XPATH,"//*[@class='fd-scrollable fd-props-scrollable']/form/fieldset/div/div/div[3]/div/div/div")
+        self.autoselectedvalue = (By.XPATH,"//ul[@class='atwho-view-ul']/li[@class='cur'] ")
+        self.autoselectedvalue1 = (By.XPATH,"//*[@id='atwho-ground-property-labelRef']//ul[@class='atwho-view-ul']/li[@class='cur']")
+        self.edit_state_table = (By.XPATH,"(//tr[td[span[text()='state']]]//button)[1]")
+        self.select_column = (By.XPATH, "(//div[div[h4[span[text()='state']]]]/div[2]/fieldset/div[4]/table/tbody/tr/td[2]/input)[1]")
+        self.savestate = (By.XPATH, "//div[div[h4[span[text()='state']]]]//button[@data-bind='click: saveEdit']")
+        self.Ecase_list = (By.XPATH, "(//*[@data-label='Edit Pen'])[1]")
+        self.errormessage = (By.XPATH, "(//*[@class='messages'])[1]")
+        self.savedbutton = (By.XPATH, "//span[text()='Saved']")
+        self.numberofquestions = (By.XPATH, "//*[@id='formdesigner']/div[1]/div[1]/div[1]/div[4]/div[1]/ul/li")
+        self.childnode = (By.XPATH, "//*[@id='formdesigner']/div[1]/div[1]/div[1]/div[4]/div[1]/ul/li/ul")
```

Comment:
Please fix the below variable names. Variable names should have _ in between 2 words, no UPPER case letters, no camelCasing.

```errorUploadmsg, errormsg, successmsg, select_hypertensioncheckbox, closedownloadpopup, erroralert_msg, selectlookuptable, lookuptableOptions, makenewbuild, inapp_registrationform, inapp_selectoption, registrationform,    SPECIFIC_Registration_form, valueerror, displayerror, autoselectedvalue, autoselectedvalue1, savestate,  Ecase_list, errormessage, savedbutton, numberofquestions, childnode```